Origami/src/components/routes.js

```diff
@@ -16,15 +16,26 @@ import SelectInputComponentsNonGHPageComponent from "./deployment/UsePrebuiltPro
 import SelectOutputComponentsNonGHPageComponent from "./deployment/UsePrebuiltProject/selectOutputComponentPage";
 import NGHDemoPageComponent from "./deployment/UsePrebuiltProject/nghDemoPage";
 import NGHDemoFrameComponent from "./deployment/UsePrebuiltProject/nghDemoFrame";
+import DefaultLayout from "../ui/src/containers/DefaultLayout" 
+import Loadable from 'react-loadable'
+
+
+function Loading() {
+  return <div>Loading...</div>;
+}
 
 export default (
   <Switch>
-    <Route exact path="/" component={HomePageComponent} />
-
+  
+    <Route  path="/" component={DefaultLayout} />
+    <Route exact path="/home" component={HomePageComponent} />
+    <Route exact path="/profile" component={LoginComponent} />
     <Route exact path="/ngh/user" component={NonGHUserProfileComponent} />
+
+
     <Route
       exact
-      path="/ngh/user/register"
+      path="/demo_register"
```

Comment:
Why?

Reply:
Hey @AvaisP  i didnt understand the use or implication of ngh and i guess its a legacy code(?).Thats why i modified the url.

Reply:
Cool